package.json

```diff
@@ -10,30 +10,30 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "gulp",
-    "watch": "gulp watch"
+    "dev": "node ./webpack.server.js",
+    "build": "rimraf ./dist && webpack --config ./webpack/webpack.prod.config.js",
+    "preview": "http-server ./dist -c-1 -o"
```

Comment:
So does this work? You could try using [simplehttp2server](https://github.com/GoogleChrome/simplehttp2server), but you still need to generate some certificates locally.